commands/build/lib/build.js

```diff
@@ -14,6 +14,7 @@ const umd = async (argv) => {
     mode: argv.mode || 'production',
     format: 'umd',
     argv,
+    inlineDynamicImports: true,
```

Comment:
This seem to alter the default build behaviour in nebula build, do we want to do that/is that necessary?

Reply:
it doesn't, since it's only actually put in the config if codeSplit is sent in as an option, but I'll change it since it's unclear

Reply:
Okey, I see 👍